tools/model-info-file.ts

```diff
@@ -0,0 +1,33 @@
+interface ModelInfoFile {
```

Comment:
Where will the official documentation for this live?

Reply:
It will live in the https://help.keyman.com/developer/cloud/ area, alongside the https://help.keyman.com/DEVELOPER/cloud/keyboard_info/1.0/ .keyboard_info file. Again, I have a PR open for this.

There is also a PR open against api.keyman.com/schemas for the corresponding JSON schema (and the updated kmp.json schema as well). Also hopefully to go public soon.

Reply:
* https://github.com/keymanapp/api.keyman.com/pull/1 covers the JSON schema and changes to package metadata format.
* https://github.com/keymanapp/help.keyman.com/pull/1 includes the documentation for the .model_info file (as of now, WIP).

These PRs are both #1 because we've just moved api and help sites to public source.